client/blocks/daily-post-button/test/index.jsx

```diff
@@ -1,43 +1,40 @@
 /** @format */
+jest.mock( 'reader/stats', () => ( {
+	pageViewForPost: () => {},
+	recordAction: () => {},
+	recordGaEvent: () => {},
+	recordTrackForPost: () => {},
+} ) );
+jest.mock( 'lib/analytics', () => ( {
```

Comment:
Not a blocker, but it may be worth it to have a blanket analytics mock that noops all of the analytics. Can we also declare it in a comment like `@jest-environment jsdom`? Or even, can we make it the default and warn developers if they call them directly in the test file (vs. in a library)?